ldk/goja/src/clipboard/clipboard.test.ts

```diff
@@ -1,5 +1,5 @@
 import { mocked } from 'ts-jest/utils';
-import { clipboard } from './clipboard';
```

Comment:
I'm not oppose, but could I ask why? I thought previous implementation allow a much cleaner scope importing.

Reply:
This was just how I was taught to do things in JS projects, with each folder having an index file and then exporting anything that needed to be from there. If that's not how we want to structure the project, then that's fine. But I personally like having more folder structure in a project so I know where to look for stuff.